From be6848c4911436e5d362e2e40ba9f8babfe0c226 Mon Sep 17 00:00:00 2001
From: Jari Aalto <jari.aalto@cante.net>
Date: Sun, 14 Oct 2012 20:37:08 +0300
Subject: [PATCH] vis/syshdrs.h: cygwin curses
Organization: Private
Content-Type: text/plain; charset="utf-8"
Content-Transfer-Encoding: 8bit


Signed-off-by: Jari Aalto <jari.aalto@cante.net>
---
 vis/syshdrs.h |    4 ++--
 1 files changed, 2 insertions(+), 2 deletions(-)

diff --git a/vis/syshdrs.h b/vis/syshdrs.h
index 1af1746..d7d4cdb 100644
--- a/vis/syshdrs.h
+++ b/vis/syshdrs.h
@@ -140,10 +140,10 @@
 
 #ifdef HAVE_NCURSES_H
 #	define NCURSES_OPAQUE 0
-#	include <ncurses.h>
+#	include <ncurses/ncurses.h>
 #else
 #	ifdef HAVE_CURSES_H
-#		include <curses.h>
+#		include <ncurses/curses.h>
 #	endif
 #endif
 
-- 
1.7.9

